lib/modules/filteReddit.js

```diff
@@ -746,9 +753,17 @@ module.always = function() {
 	Menu.addMenuItem(nsfwSwitch, onClickSwitch);
 };
 
+let filterline;
+export let forceExpandoCompletion = false;
```

Comment:
I don't really think this belongs in filteReddit. I'd prefer to have showImages export a function allowing the caller to enable/disable forced expando completion.